print.php

```diff
@@ -1,16 +1,15 @@
-<?php
+<?
```

Comment:
We've been using <?php for compatibility. Short hand is widely debated, and often disabled. For our purposes we can make sure its on. Since this is the second time it;s come up ;-)

Reply:
I purposely did a find / replace to do that - my mistake :S